test/integration/userStatus.test.js

```diff
@@ -554,6 +556,135 @@ describe("UserStatus", function () {
     });
   });
 
+  describe("PATCH /users/status/self", function () {
+    let userId;
+    let userJwt;
+
+    before(async function () {
+      userId = await addUser(userData[8]);
+      userJwt = authService.generateAuthToken({ userId });
+    });
+
+    afterEach(async function () {
+      await firestore.collection("tasks").doc("user1AssignedTask").delete();
+      await firestore.collection("usersStatus").doc("user1AssignedStatus").delete();
+    });
+
+    after(async function () {
+      await firestore.collection("users").doc(userId).delete();
+    });
+
+    it("Should Change the status to ACTIVE if user has task assigned.", async function () {
+      const now = new Date();
+      const nowTimeStamp = new Date().setUTCHours(0, 0, 0, 0);
+      const fiveDaysFromNowTimeStamp = new Date(now.setUTCHours(0, 0, 0, 0) + 5 * 24 * 60 * 60 * 1000);
```

Comment:
all these can be set as constant in the `describe` body outside the it-block. That will reduce code duplication

Reply:
The variables declared in this `describe` block are only needed in two of the `it` blocks. To avoid unnecessary clutter, I have moved the variables to the specific `it` blocks where they are needed. Let me know if you are ok with this